### PR TITLE
Replace Travis CI by GitHub Actions

### PR DESCRIPTION
**Pull request by hartwork**
I'll add support for Python >=3.8 in a follow-up PR…

### CODE REVIEW

**Comment by hartwork** [comment]
PS: I'm happy to add support for macOS in a follow-up PR as well.

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - uses: actions/setup-python@v2.1.4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install build dependencies
+      run: |
+        sudo apt-get install --no-install-recommends -y graphviz
+        pip install -r test_requirements.txt
+    - name: Run the test suite
+      run: |
+        git config --global user.email tests@example.com
```
Comment (esc) [inline .github/workflows/run-tests.yml:42]:
```suggestion
        git config --global user.email tests@git-big-picture.test
```

https://en.m.wikipedia.org/wiki/.test

**Review by esc — CHANGES_REQUESTED** [review]
Looks good! One suggestion for your consideration.

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (hartwork) [reply .github/workflows/run-tests.yml:42]: Okay interesting, why not :smiley: Will repush in a minute.

**Review by esc — APPROVED** [review]

**Comment by esc** [comment]
🔥

**Comment by esc** [comment]
FWIW: I removed the Travis integration - however, since the repo had been moved already, I had to give Travis access to the organization temporarily. I have since revoked that access again.

**Pull request merged**